packages/edit-post/src/components/sidebar/style.scss

```diff
@@ -1,3 +1,7 @@
+.block-editor-editor-skeleton__sidebar > div {
```

Comment:
I'm not sure this is actually better. There is NO way around this, short of removing the fragment entirely, or giving it a CSS class to target separately, we _have_ to target this div. 

Is this better, @youknowriad?

Reply:
I think that's better, I have a small preference for this being moved to the layout style but it's fine here.

Reply:
Let me move it to the layout style, that's no trouble. It's not important to me where it lives, just that it's present :)

Reply:
Moved, let me know if it's still okay to go.